casepro/backend/junebug.py

```diff
@@ -463,3 +467,42 @@ def received_junebug_message(request):
         created_on=datetime.now(), has_labels=True)
 
     return JsonResponse(msg.as_json())
+
+
+@csrf_exempt
+def receive_identity_store_optout(request):
```

Comment:
We probably want to require a login / authentication and authorization of some sort here if we can.